foundry.toml

```diff
@@ -6,7 +6,8 @@ solc_version = '0.8.25'
 evm_version = 'shanghai'
 test = 'tests'
 bytecode_hash = "none"
-auto_detect_remappings = false
+# TODO: this fails to compile when set to false
+auto_detect_remappings = true
```

Comment:
We should do a quick investigation on this. It might be because ICTT no longer has multiple copies of some submodules